frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.unit.spec.tsx

```diff
@@ -174,14 +175,28 @@ describe("Visualizations > PivotTable > PivotTable", () => {
 
     setup({ initialSettings: hiddenSettings });
 
-    rows.forEach(rowData => {
-      expect(screen.getByText(rowData[0].toString())).toBeInTheDocument();
+    expect(screen.getByText("Totals for bar1")).toBeInTheDocument();
 
-      [1, 2, 4, 5].forEach(colIndex => {
-        expect(
-          screen.queryByText(rowData[colIndex].toString()),
-        ).not.toBeInTheDocument();
-      });
+    expect(screen.getByRole("img", { name: /add/i })).toBeInTheDocument();
```

Comment:
I'm not sure I have a good idea about what that element is. Is there anything we can do to make it a bit more straightforward?

Reply:
Could add a small comment. It's just asserting that the + is present so that the collapsed section can be expanded